src/coreclr/nativeaot/Runtime/EHHelpers.cpp

```diff
@@ -449,6 +449,18 @@ int32_t __stdcall RhpHardwareExceptionHandler(uintptr_t faultCode, uintptr_t fau
 
 #else // TARGET_UNIX
 
+static bool g_ContinueSearchOnUnhandledRuntimeException = false;
+
+// Set the runtime to continue search when encountering an unhandled runtime exception. Once done it is forever.
+// Continuing the search allows any vectored exception handlers or SEH installed by the client to take effect.
+// Any client that does so is expected to handle stack overflows.
+EXTERN_C bool RhpContinueSearchOnUnhandledRuntimeException()
```

Comment:
```suggestion
EXTERN_C void RhpContinueOnFatalErrors()
```
The previous value is not really useful since the setting is applied forever. Also, I think fatal error better describes what this does.

Reply:
The return allows you to realize you did a second call to no effect, but I agree that this is not really useful.
I like 'FatalError' better than 'ContinueSearchOnUnh....'. It is also much shorter! I'll change the naming of the setter and the variable it acts on.